parsing/src/main/resources/Parser.jj

```diff
@@ -1333,6 +1334,11 @@ SqlSelect SqlSelect() :
         keywordList = new SqlNodeList(keywords, s.addAll(keywords).pos());
     }
     selectList = SelectList()
+<#if parser.includeExceptExpression>
+    (
```

Comment:
super nit: while they have the same behavior, the convention in this function appears to be to use brackets rather than parens with a "?".